src/com/palmergames/bukkit/towny/TownySettings.java

```diff
@@ -2083,25 +2084,24 @@ public static int getMaxResidentsPerTown() {
 	}
 
 	public static boolean isTownyUpdating(String currentVersion) {
-
-		if (isTownyUpToDate(currentVersion))
-			return false;
-		else
-			return true; // Assume
+		// Assume
+		return !isTownyUpToDate(currentVersion);
 	}
 
 	public static boolean isTownyUpToDate(String currentVersion) {
-
-		return currentVersion.equals(getLastRunVersion(currentVersion));
+		return currentVersion.equals(currentVersion);
```

Comment:
This line has to go back to `return currentVersion.equals(getLastRunVersion(currentVersion));` or Towny will never run the Towny#update() code.